lib/ExtraCorres.thy

```diff
@@ -278,17 +278,16 @@ lemma wp_from_corres_unit:
   f' \<lbrace>\<lambda>_ s'. \<exists>s. (s,s') \<in> state_relation \<and> Q s \<and> Q' s'\<rbrace>"
   by (auto intro!: wp_from_corres_u_unit)
 
-lemma corres_whileLoop_results:
+lemma corres_whileLoop_results_helper:
```

Comment:
Out of curiosity, what has changed that this is now a helper?

Reply:
This was in restoring the original version of the `corres_whileLoop` proofs

Reply:
(The diff is a bit hard to read. If you open the source for this file, it's easy to see that it's just the old version plus the new stuff for abstract termination and `no_fail`.)